Bottom: aad0eb0a160fdf4a6b51a39cab2586aa355a5a0a
Top:    7fd80449b6e3ff3efb57bba0e98e2196d5aa5bd8
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-06 13:40:50 +0800

Refresh of autofs-5.0.5-fix-master-map-bogus-keywork-match.patch

---

diff --git a/lib/master_tok.l b/lib/master_tok.l
index 6ff18c0..064f8fd 100644
--- a/lib/master_tok.l
+++ b/lib/master_tok.l
@@ -91,7 +91,7 @@ OPTWS		[[:blank:]]*
 NL		\r?\n
 CONT		\\\n{OPTWS}
 
-OPTIONSTR	([\-]?([[:alpha:]_]([[:alnum:]_\-])*(="?([[:alnum:]_\-\:])+\"?)*)+)
+OPTIONSTR	([\-]?([[:alpha:]_]([[:alnum:]_\-])*(=(["]?([[:alnum:]_\-\:])+["]?)*)+))
 MACROSTR	(-D{OPTWS}([[:alpha:]_]([[:alnum:]_\-\.])*)=([[:alnum:]_\-\.])+)
 SLASHIFYSTR	(--(no-)?slashify-colons)
 NUMBER		[0-9]+
